buttercup.el

```diff
@@ -4,6 +4,7 @@
 
 ;; Version: 1.7
 ;; Author: Jorgen Schaefer <contact@jorgenschaefer.de>
+;; Package-Requires: ((emacs "24.4"))
```

Comment:
This is incorrect. buttercup works with Emacs 24.3 (and even older versions, really, just not tested).

Reply:
lexical-binding, as declared on top of the file, requires at least Emacs 24. `autoloadp`, as used in `spy-on`, requires at least 24.3. I fixed it to 24.3. Do you want to add `autoloadp` to compat?